dcos_installer/upgrade.py

```diff
@@ -50,12 +50,20 @@
 fi
 
 SKIP_CHECKS=false
+VERBOSE=false
 
 if [[ $# -ne 0 ]]; then
-  if [[ "$1" = "--skip-checks" ]]; then
-     echo "Skipping checks"
-     SKIP_CHECKS=true
-  fi
+    for var in "$@";
+    do
```

Comment:
This `do` should be on the preceding line in order to be consistent with the rest of the script.